worlds/shivers/Options.py

```diff
@@ -1,6 +1,11 @@
-from Options import Choice, DefaultOnToggle, Toggle, PerGameCommonOptions, Range
 from dataclasses import dataclass
 
+from Options import (
+    Choice, DefaultOnToggle, ItemDict, ItemSet, LocationSet, OptionGroup, PerGameCommonOptions, Range, Toggle,
+)
+from worlds.shivers import ItemType, item_table
+from worlds.shivers.Constants import location_info
```

Comment:
These need to be relative imports
```suggestion
from . import ItemType, item_table
from .Constants import location_info
```